test/e2e/test_job_submissions.py

```diff
@@ -32,13 +32,21 @@
 import os
 import configparser
 import tempfile
-from e2e.utils import wait_for_job_output, submit_sleep_job, submit_custom_job
+from e2e.utils import (
+    wait_for_job_output,
+    submit_sleep_job,
+    submit_custom_job,
+    submit_job_from_bundle,
+    verify_output_dir_matches,
+)
 
 
 LOG = logging.getLogger(__name__)
 
 
 class TestJobSubmission:
+    JOB_OUTPUT_PATH = os.path.join(os.getcwd(), "job_output")
```

Comment:
nit: any reason we can't use the [tmp_path fixture](https://docs.pytest.org/en/6.2.x/tmpdir.html) here and make the files in there?